go/src/koding/klientctl/endpoint/machine/info.go

```diff
@@ -0,0 +1,130 @@
+package machine
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"koding/klient/machine"
+)
+
+// Info stores the basic information about the machine.
+type Info struct {
+	// ID is an unique identifier for a given machine.
+	ID string `json:"id"`
+
+	// Alias stores a human readable alias for machine.
+	Alias string `json:"alias"`
+
+	// Team is the name of the team which created the machine.
+	Team string `json:"team"`
+
+	// Stack describes machine parent stack.
+	Stack string `json:"stack"`
+
+	// Provider represents machine provider.
+	Provider string `json:"provider"`
+
+	// Label is the machine label, as seen by the Koding UI.
+	Label string `json:"label"`
+
+	// The IP of the running machine.
+	IP string `json:"ip"`
+
+	// QueryString is string representation of kite protocol query.
+	QueryString string `json:"query_string"`
```

Comment:
The json here should be camel-case `queryString`.

We only use snake-case in kloud providers, mostly for backward-compatibility (and because we store Terraform's attributes directly in mongo in some cases).

All new code should use snake-case (except where it's still required - see kloud providers), and in future we should streamline this to use snake-case everywhere.